packages/terra-search-field/src/SearchField.jsx

```diff
@@ -34,12 +39,19 @@ const propTypes = {
    * A callback to indicate an invalid search. Sends parameter {String} searchText.
    */
   onInvalidSearch: PropTypes.func,
+
+  /**
+   * How the component should display from a fixed size to stretching to fill its container
+   */
+  sizing: PropTypes.oneOf(['fixed', 'stretch']),
```

Comment:
This goes back to discussion on the issue @StephenEsser & @Singleton06 had regarding this prop. Was it ever determined if there would be more than one sizing option other than 'stretch'? 'Fixed' doesn't apply any special logic, and IMO the behavior 'fixed' is representing here is actual 'auto'. To me, 'fixed' would indicate an max-width was attached to the search field.